### ISSUE TITLE
Add the ability to modify the style and position for the thumbnail container

### ISSUE THREAD

**Issue by alanatkinson** [ansu5555/pdf-viewer-reactjs#52]
**Is your feature request related to a problem? Please describe.**
I need to change the styling for the container - in particular the background color. 

**Describe the solution you'd like**
You've helpfully provided the ability to get to the viewer wrapper, canvas wrapper and to either style or override the navigation controls (nice work on the latter btw). I'd like two more properties thumbnailCss and thumbnailsOnTop that work like canvasCss and navbarOnTop respectively

**Describe alternatives you've considered**
Looking at the code I can't see a way to easily get to these containers to modify them without hacking at it.

**Additional context**
Shouldn't need anything - it's a fairly simple request in concept.

**Comment by DhruvikBarvaliya** [comment]
ok

**Comment by ansu5555** [comment]
Implemented in v2.2.1

Added more customization options, you can now provide customization to thumbnail using below prop

**`showThumbnail`:**

-   Type:

```js
PropTypes.shape({
    scale: PropTypes.number, // Thumbnail scale, ranges from 1 to 5

    rotationAngle: PropTypes.number, // Thumbnail rotation angle, values can be -90, 0 or 90. Default is 0

    onTop: PropTypes.bool, // Thumbnail position, if set to true thumbnail will be placed on top

    backgroundColor: PropTypes.string, // Color(hex or rgb) of the thumbnail container

    thumbCss: PropTypes.string, // Custom css class for thumbnails

    selectedThumbCss: PropTypes.string, // Custom css class for selected thumbnail
})
```
